README.md

```diff
@@ -397,10 +397,16 @@ render conn, data: data, opts: [meta: meta_data]
 
 ### Key Format (for Attribute, Relationship and Query Param)
 
-By default keys are `dash-erized` as per the jsonapi.org recommendation, but
-keys can be customized via config.
+By default keys are `dash-erized` as per the JSON:API 1.0 recommendation, but keys can be customized via config.
 
-In your `config.exs` file:
+In your `config.exs` file you can use `camelcased` recommended by upcomming JSON:API 1.1:
+
+```elixir
+config :ja_serializer,
+  key_format: :camelcased
+```
+
+Or `underscored` to keep the keys same as Elixir conventions:
```

Comment:
I'm not sure I'd say underscored attribute keys are really an Elixir convention since it's really the client that would care about this. I would just cut this to "Or `underscored`: "

Reply:
Yep, no problem. But naming in Elixir really does follow underscored for keys IMHO.